### DISCUSSION TITLE
Bug: Discussion detail view doesn't show labels

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#13011] [Mobile] [answered]
In the discussion overview one can see labels, but if you click on a specific discussion you don't see any labels.

## Overview
![IMG_20220318_120624.jpg](https://user-images.githubusercontent.com/20150243/158992502-0f8501d7-7aab-4d71-9efc-ca2ed8d1d36b.jpg)

## Details
![Screenshot_2022-03-18-12-06-33-248_com.github.android.jpg](https://user-images.githubusercontent.com/20150243/158992522-19424497-1ca4-42ff-9299-0e167dfe9815.jpg)

**Comment by aconsuegra** [comment, marked as answer]
Hi @Zerotask, thanks for reaching out with your feedback! You should be able to see (and edit if you have permissions) the labels if you tap on the `i` button in the bottom right corner. 

this is consistent with issues and pull requests, so would you expect to see them somewhere else too?

**Reply by Zerotask** [reply]
Oh, you're right. Thanks for pointing that out. Then this discussion can be closed.
